src/types.ts

```diff
@@ -457,6 +457,18 @@ function genEnum(
       isExported: true,
       name: variant.name,
       properties: [
+        {
```

Comment:
It looks like this adds new static members instead of a replacing the instance variables - i.e. there are now both static and instance vars. Is this intentional?

Reply:
Yes, that's intentional. In typescript you cannot access static fields on an instance. So if we had only static fields you wouldn't be able to do `foo.kind`, only `Foo.kind`. So I have added both so we can do both.